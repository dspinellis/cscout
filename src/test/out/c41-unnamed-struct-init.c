Table: Ids
EID	NAME	READONLY	UNDEFMACRO	MACRO	MACROARG	ORDINARY	SUETAG	SUMEMBER	LABEL	TYPEDEF	ENUM	YACC	FUN	CSCOPE	LSCOPE	UNUSED
91	MAXDIGIT	0	0	1	0	0	0	0	0	0	0	0	0	0	0	1
331	x	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
441	main	1	0	0	0	1	0	0	0	0	0	0	1	0	1	0
571	qqq	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
619	apoint	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
759	first	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
891	foo	0	0	0	0	1	0	0	0	0	0	0	1	0	1	1
1019	x	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1279	y	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1519	z	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1679	s	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
1799	last	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
1879	ap	0	0	0	0	1	0	0	0	0	0	0	0	0	1	0
1999	point	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
2259	x	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
2319	u	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
2369	p	0	0	0	0	1	0	0	0	0	0	0	0	0	1	0
2975	__DATE__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
3295	__TIME__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
3575	__FILE__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
3865	__LINE__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
4055	__STDC__	1	0	1	0	0	0	0	0	0	0	0	0	0	0	1
5355	_cscout_dummy1	1	0	0	0	1	0	0	0	0	0	0	1	1	0	0
Table: Tokens
FID	FOFFSET	EID
2	297	2975
2	329	3295
2	357	3575
2	386	3865
2	405	4055
2	471	441
2	535	5355
2	558	5355
4	61	619
4	75	759
4	101	1019
4	127	1279
4	151	1519
4	167	1679
4	179	1799
4	187	1879
4	199	1999
4	225	2259
4	231	2319
4	236	2369
4	244	441
4	254	1879
4	257	1019
4	273	619
4	285	1019
4	294	1279
4	322	619
4	334	1019
4	353	619
4	365	1279
4	384	619
4	396	1519
4	415	619
4	427	1279
4	456	619
4	468	1519
4	486	2369
4	488	2319
4	490	2259
4	506	1999
4	519	2259
4	541	1999
4	552	2319
4	559	2259
5	8	91
5	32	331
5	43	441
5	56	571
5	88	891
Table: Rest
FID	FOFFSET	CODE
2	287	\u0000a\u0000a#define 
2	305	  
2	320	\u0000a#define 
2	337	 
2	348	\u0000a#define 
2	365	 
2	377	\u0000a#define 
2	394	 1\u0000a#define 
2	413	 1\u0000a\u0000a
2	466	\u0000aint 
2	475	();\u0000a
2	522	\u0000astatic void 
2	549	(void) { 
2	572	(); }\u0000a
3	152	\u0000a\u0000a\u0000a#pragma includepath 
3	195	\u0000a
3	239	\u0000astatic void _cscout_dummy2(void) { _cscout_dummy2(); }\u0000a
4	53	\u0000astruct 
4	67	 {\u0000a	int 
4	80	;\u0000a	struct {\u0000a		double 
4	102	;\u0000a	};\u0000a	struct {\u0000a		double 
4	128	;\u0000a		struct {\u0000a			double 
4	152	;\u0000a		};\u0000a		char *
4	168	;\u0000a	};\u0000a	int 
4	183	;\u0000a} 
4	189	;\u0000a\u0000astruct 
4	204	 {\u0000a	union {\u0000a		double 
4	226	;\u0000a	} 
4	232	;\u0000a} 
4	237	;\u0000a\u0000aint 
4	248	() {\u0000a	
4	256	.
4	258	 = 3;\u0000a	(struct 
4	279	) {  .
4	286	 = 42, .
4	295	 = 12, 9, 
4	309	 };\u0000a	(struct 
4	328	) {  .
4	335	 = 42 };\u0000a	(struct 
4	359	) {  .
4	366	 = 42 };\u0000a	(struct 
4	390	) {  .
4	397	 = 42 };\u0000a	(struct 
4	421	) {  .
4	428	 = 42, 12, 
4	443	 };\u0000a	(struct 
4	462	) {  .
4	469	 = 42, 
4	480	 };\u0000a\u0000a	
4	487	.
4	489	.
4	491	 = 3;\u0000a	(struct 
4	511	) {  { .
4	520	 = 42 }  };\u0000a	(struct 
4	546	) {  .
4	553	 = { .
4	560	 = 42 }  };\u0000a}\u0000a
5	0	#define 
5	16	 11\u0000d\u0000aextern int 
5	33	;\u0000d\u0000aextern 
5	47	();\u0000d\u0000aint 
5	59	;\u0000d\u0000a\u0000d\u0000a
5	86	\u0000d\u0000a
5	91	() {\u0000d\u0000a	
5	116	}\u0000d\u0000a\u0000d\u0000a
Table: Projects
PID	NAME
16	unspecified
17	Prj1
18	Prj2
Table: IdProj
EID	PID
441	17
619	17
759	17
1019	17
1279	17
1519	17
1679	17
1799	17
1879	17
1999	17
2259	17
2319	17
2369	17
2975	17
3295	17
3575	17
3865	17
4055	17
5355	17
91	18
331	18
441	18
571	18
619	18
759	18
891	18
1019	18
1279	18
1519	18
1679	18
1799	18
1879	18
1999	18
2259	18
2319	18
2369	18
2975	18
3295	18
3575	18
3865	18
4055	18
5355	18
Table: Files
FID	NAME	RO	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	NSTRING	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NPPTOKEN	NCTOKEN	NCOPIES	NSTATEMENT	NPFUNCTION	NFFUNCTION	NPVAR	NFVAR	NAGGREGATE	NAMEMBER	NENUM	NEMEMBER	NINCFILE
2	host-defs.h	1	578	367	29	0	3	22	61	3	0	5	0	0	5	37	18	1	1	0	1	0	0	0	0	0	0	0
3	host-incs.h	1	295	187	13	0	2	13	54	1	0	1	0	0	0	16	0	1	0	0	0	0	0	0	0	0	0	0
4	c41-unnamed-struct-init.c	0	574	49	164	0	1	37	48	3	0	0	0	0	0	181	182	1	10	1	0	2	0	6	8	0	0	0
5	prj2.c	0	121	34	26	2	0	11	21	0	0	1	0	0	1	21	18	1	0	1	0	2	0	0	0	0	0	0
Table: FileProj
FID	PID
2	17
3	17
4	17
1	18
2	18
3	18
4	18
5	18
Table: Definers
PID	CUID	BASEFILEID	DEFINERID
17	4	4	2
18	4	4	2
18	5	5	2
Table: Includers
PID	CUID	BASEFILEID	INCLUDERID
17	2	2	1
17	4	3	1
17	4	4	1
18	2	2	1
18	2	2	1
18	4	3	1
18	4	4	1
18	5	3	1
18	5	5	1
Table: Providers
PID	CUID	PROVIDERID
17	2	2
17	4	4
18	2	2
18	2	2
18	4	4
18	5	5
Table: IncTriggers
PID	CUID	BASEFILEID	DEFINERID	FOFFSET	LEN
17	4	4	2	471	4
18	4	4	2	471	4
18	5	5	2	471	4
Table: Functions
ID	NAME	ISMACRO	DEFINED	DECLARED	FILESCOPED	FID	FOFFSET	FANIN
891	foo	0	1	1	0	5	88	0
2449	main	0	1	1	0	4	244	0
5355	_cscout_dummy1	0	1	1	1	2	535	1
Table: FunctionMetrics
FUNCTIONID	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	NSTRING	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NPPTOKEN	NCTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NGNSOC	NPARAM	MAXNEST	NLABEL	FANIN	FANOUT	CCYCL1	CCYCL2	CCYCL3	CSTRUC	CHAL	IFLOW	FIDBEGIN	FOFFSETBEGIN	FIDEND	FOFFSETEND
891	23	15	4	1	0	2	18	0	0	0	0	0	0	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	18	0	0	0	0	0	1	1	1	0.0	0.0	0.0	5	95	5	117
2449	322	0	99	0	0	13	48	3	0	0	0	0	0	125	125	10	25	2	13	0	0	0	0	0	0	0	0	0	0	0	0	0	2	0	0	2	2	0	0	2	13	0	0	0	0	0	1	1	1	0.0	163.499	0.0	4	252	4	573
5355	21	0	3	0	0	1	20	0	0	0	0	0	0	5	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	1	0	1	10	0	0	0	1	1	1	1	1	1.0	0.0	1.0	2	557	2	577
Table: FunctionId
FUNCTIONID	ORDINAL	EID
891	0	891
2449	0	441
5355	0	5355
Table: Fcalls
SOURCEID	DESTID
5355	5355
Done
