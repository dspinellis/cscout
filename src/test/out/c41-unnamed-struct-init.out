Table: Ids
EID	NAME	READONLY	UNDEFMACRO	MACRO	FUNMACRO	MACROARG	CPPCONST	CPPSTRVAL	DEFCCONSTVAL	NOTDEFCCONSTVAL	EXPCCONSTVAL	NOTEXPCCONSTVAL	ORDINARY	SUETAG	SUMEMBER	LABEL	TYPEDEF	ENUM	YACC	FUN	CSCOPE	LSCOPE	UNUSED
91	MAXDIGIT	0	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
331	x	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
441	main	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
571	qqq	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
619	apoint	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
759	first	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
891	foo	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	1
1019	x	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1279	y	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1519	z	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
1679	s	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
1799	last	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
1879	ap	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	0
1999	point	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
2259	x	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
2319	u	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
2369	p	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	0
2975	__DATE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3295	__TIME__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3575	__FILE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3865	__LINE__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
4055	__STDC__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
5355	_cscout_dummy1	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	0
Table: Tokens
FID	FOFFSET	EID
2	297	2975
2	329	3295
2	357	3575
2	386	3865
2	405	4055
2	471	441
2	535	5355
2	558	5355
4	61	619
4	75	759
4	101	1019
4	127	1279
4	151	1519
4	167	1679
4	179	1799
4	187	1879
4	199	1999
4	225	2259
4	231	2319
4	236	2369
4	244	441
4	254	1879
4	257	1019
4	273	619
4	285	1019
4	294	1279
4	322	619
4	334	1019
4	353	619
4	365	1279
4	384	619
4	396	1519
4	415	619
4	427	1279
4	456	619
4	468	1519
4	486	2369
4	488	2319
4	490	2259
4	506	1999
4	519	2259
4	541	1999
4	552	2319
4	559	2259
5	8	91
5	32	331
5	43	441
5	56	571
5	88	891
Table: Rest
FID	FOFFSET	CODE
2	287	\u0000a\u0000a#define 
2	305	  
2	320	\u0000a#define 
2	337	 
2	348	\u0000a#define 
2	365	 
2	377	\u0000a#define 
2	394	 1\u0000a#define 
2	413	 1\u0000a\u0000a
2	466	\u0000aint 
2	475	();\u0000a
2	522	\u0000astatic void 
2	549	(void) { 
2	572	(); }\u0000a
3	152	\u0000a\u0000a\u0000a#pragma includepath 
3	195	\u0000a
3	239	\u0000astatic void _cscout_dummy2(void) { _cscout_dummy2(); }\u0000a
4	53	\u0000astruct 
4	67	 {\u0000a	int 
4	80	;\u0000a	struct {\u0000a		double 
4	102	;\u0000a	};\u0000a	struct {\u0000a		double 
4	128	;\u0000a		struct {\u0000a			double 
4	152	;\u0000a		};\u0000a		char *
4	168	;\u0000a	};\u0000a	int 
4	183	;\u0000a} 
4	189	;\u0000a\u0000astruct 
4	204	 {\u0000a	union {\u0000a		double 
4	226	;\u0000a	} 
4	232	;\u0000a} 
4	237	;\u0000a\u0000aint 
4	248	() {\u0000a	
4	256	.
4	258	 = 3;\u0000a	(struct 
4	279	) {  .
4	286	 = 42, .
4	295	 = 12, 9, 
4	309	 };\u0000a	(struct 
4	328	) {  .
4	335	 = 42 };\u0000a	(struct 
4	359	) {  .
4	366	 = 42 };\u0000a	(struct 
4	390	) {  .
4	397	 = 42 };\u0000a	(struct 
4	421	) {  .
4	428	 = 42, 12, 
4	443	 };\u0000a	(struct 
4	462	) {  .
4	469	 = 42, 
4	480	 };\u0000a\u0000a	
4	487	.
4	489	.
4	491	 = 3;\u0000a	(struct 
4	511	) {  { .
4	520	 = 42 }  };\u0000a	(struct 
4	546	) {  .
4	553	 = { .
4	560	 = 42 }  };\u0000a}\u0000a
5	0	#define 
5	16	 11\u0000d\u0000aextern int 
5	33	;\u0000d\u0000aextern 
5	47	();\u0000d\u0000aint 
5	59	;\u0000d\u0000a\u0000d\u0000a
5	86	\u0000d\u0000a
5	91	() {\u0000d\u0000a	
5	116	}\u0000d\u0000a\u0000d\u0000a
Table: Projects
PID	NAME
23	unspecified
24	Prj1
25	Prj2
Table: IdProj
EID	PID
441	24
619	24
759	24
1019	24
1279	24
1519	24
1679	24
1799	24
1879	24
1999	24
2259	24
2319	24
2369	24
2975	24
3295	24
3575	24
3865	24
4055	24
5355	24
91	25
331	25
441	25
571	25
619	25
759	25
891	25
1019	25
1279	25
1519	25
1679	25
1799	25
1879	25
1999	25
2259	25
2319	25
2369	25
2975	25
3295	25
3575	25
3865	25
4055	25
5355	25
Table: Files
FID	NAME	RO
2	host-defs.h	1
3	host-incs.h	1
4	c41-unnamed-struct-init.c	0
5	prj2.c	0
Table: Filemetrics
FID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NCOPIES	NINCFILE	NPFUNCTION	NFFUNCTION	NPVAR	NFVAR	NAGGREGATE	NAMEMBER	NENUM	NEMEMBER
2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	16	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	2	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	1	0	0	0	0	0	0
2	1	548	367	29	0	3	22	61	47	NULL	1	1	0	0	0	5	0	0	5	37	2	0	0	2	0	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	2	5	8	1	1	5	7	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
3	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	15	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	0	0	0	0	0	0	0
3	1	243	187	13	0	2	13	48	28	NULL	1	1	0	0	0	1	0	0	0	16	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
4	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	32	0	3	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	182	23	26	0	13	0	3	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	10	0	NULL	10	3	0	NULL	3	0	NULL	NULL	NULL	1	0	2	0	6	8	0	0
4	1	502	49	164	0	1	37	46	33	NULL	3	1	0	0	0	0	0	0	0	181	23	26	0	13	0	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	5	0	0	5	3	0	0	3	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
5	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	8	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	3	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	NULL	4	4	0	NULL	4	0	NULL	NULL	NULL	1	0	2	0	0	0	0	0
5	1	106	34	26	2	0	11	21	14	NULL	1	1	0	0	0	1	0	0	1	21	3	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	1	5	4	0	1	5	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
Table: FileProj
FID	PID
2	24
3	24
4	24
1	25
2	25
3	25
4	25
5	25
Table: Definers
PID	CUID	BASEFILEID	DEFINERID
24	4	4	2
25	4	4	2
25	5	5	2
Table: Includers
PID	CUID	BASEFILEID	INCLUDERID
24	2	2	1
24	4	3	1
24	4	4	1
25	2	2	1
25	2	2	1
25	4	3	1
25	4	4	1
25	5	3	1
25	5	5	1
Table: Providers
PID	CUID	PROVIDERID
24	2	2
24	4	4
25	2	2
25	2	2
25	4	4
25	5	5
Table: IncTriggers
PID	CUID	BASEFILEID	DEFINERID	FOFFSET	LEN
24	4	4	2	471	4
25	4	4	2	471	4
25	5	5	2	471	4
Table: Functions
ID	NAME	ISMACRO	DEFINED	DECLARED	FILESCOPED	FID	FOFFSET	FANIN
891	foo	0	1	1	0	5	88	0
2449	main	0	1	1	0	4	244	0
5355	_cscout_dummy1	0	1	1	1	2	535	1
Table: FunctionDefs
FUNCTIONID	FIDBEGIN	FOFFSETBEGIN	FIDEND	FOFFSETEND
891	5	95	5	117
2449	4	252	4	573
5355	2	557	2	577
Table: FunctionMetrics
FUNCTIONID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NGNSOC	NMPARAM	NFPARAM	NEPARAM	FANIN	FANOUT	CCYCL1	CCYCL2	CCYCL3	CSTRUC	CHAL	IFLOW
891	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	18	NULL	0	NULL	0	0	1	1	1	0.0	0.0	0.0
891	1	23	15	4	1	0	2	18	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
2449	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	32	0	2	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	125	10	25	2	13	0	3	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	2	0	NULL	2	2	0	NULL	2	0	NULL	13	NULL	0	NULL	0	0	1	1	1	0.0	163.499	0.0
2449	1	295	0	99	0	0	13	44	33	NULL	2	1	0	0	0	0	0	0	0	125	10	25	2	13	0	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	2	0	0	2	2	0	0	2	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	163.499	0.0
5355	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	0	0	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	NULL	1	0	1	NULL	1	0	NULL	10	NULL	0	NULL	1	1	1	1	1	1.0	0.0	1.0
5355	1	22	0	3	0	0	1	21	5	NULL	0	1	0	0	0	0	0	0	0	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	1	0	1	0	0	NULL	0	NULL	0	1	1	1	1	1	1.0	0.0	1.0
Table: FunctionId
FUNCTIONID	ORDINAL	EID
891	0	891
2449	0	441
5355	0	5355
Table: Fcalls
SOURCEID	DESTID
5355	5355
Done
